src/DSP/CombFilter.cpp

```diff
@@ -31,6 +33,13 @@ CombFilter::~CombFilter(void)
 {
     memory.dealloc(input);
     memory.dealloc(output);
+
+    if(Plpf != 127) { //LowPass
+        memory.dealloc(lpf);
+    }
+    if(Phpf != 0) { //HighPass
+        memory.dealloc(hpf);
+    }
```

Comment:
Minor, but IIRC dealloc will be a no-op if the input value is `nullptr` and it will replace the old pointer with `nullptr` when actually performing a dealloc, so the conditions shouldn't be needed?

Reply:
tried it without. seems to work